gcc/testsuite/rust/execute/torture/sinf32.rs

```diff
@@ -0,0 +1,22 @@
+/* { dg-output "0.893996" }*/
```

Comment:
I agree that this is probably a little sketch on some architectures :D We could lower the precision, because what matters in the end is that the intrinsic gets called. So instead we could have something like the following for the format string:
```rust
let a  = "%.3f\n\0";
```

Reply:
I changed the dg-output to just match the 3 decimals seems to work.

Reply:
I suggest to avoid adding all such execution tests -- won't this quickly "proliferate"?

Such builtins typically are mapped to regular function calls (which we already do have confidence that they work correctly), so it ought to suffice to just scan '-fdump-tree-original' (see #660).  I've prepared a commit to "Turn 'rust/execute/torture/sinf32.rs', 'rust/execute/torture/sqrtf32.rs' into 'rust/compile/torture/intrinsics-1.rs'" -- if you agree, want me to push that to this branch here?

Reply:
That sounds like a much better plan @tschwinge lets do it!

Reply:
this looks much better lets merge it :D